modules/configuration/ajax/process.php

```diff
@@ -108,7 +108,7 @@
                 }
             }
             // Add the new setting
-            $DB->insert(
+            $DB->unsafeInsert(
```

Comment:
is this okay to change for all update/inserts on the config module, not knowing what other front-end values this might affect?

Reply:
This is actually just because the PR is based on #8759 .. can you move your comment over there? I added the blocked tag to this PR.

Reply:
actually now that I think of it they're independent, so I removed the commit from this PR and I'm removing the blocked tag.. but can you still move your comment to the other PR?